sample-compose/app/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,7 @@
 <resources>
     <string name="app_name">Sample Compose</string>
     <string name="error_generic">Unexpected error</string>
+    <string name="id_title">ID: %1$d</string>
+    <string name="home_title">Home</string>
+    <string name="second_title">Second</string>
```

Comment:
Let's rename them a bit to make it clearer. Also, add one more space in each section of screen that could be nice 🚀 
```suggestion
    <string name="error_generic">Unexpected error</string>
    
    <string name="home_title_bar">Home</string>
    
    <string name="second_title_bar">Second</string>
    <string name="second_id_title">ID: %1$d</string>
```